### ISSUE TITLE
Parallel rendering 

### ISSUE THREAD

**Issue by siwel** [mifi/editly#186]
Hey,

I see with https://github.com/mifi/reactive-video we have Parallel rendering 🔥 

Would it be possible to bring this to editly, what would be involved in getting this to work?

Thanks,
Lewis

**Comment by desis123** [comment]
How about rendering each clip parallel and then join will work ?

**Comment by mifi** [comment]
Yes you could try that, but you’d have to split/merge the file yourself

**Comment by desis123** [comment]
Yes I am using it for a project text2video generator .. so basically all my clips are seperated by audio segments generated from text .. so what I am planing create multiple clips based on those audios then merge them together. While I will merge them I will use transitions .. so basically for the first part of the process I could use reactive or remotion .. but currently I am more comportable with json .. rather than create some built in templates for reactve or remoton .. may be in future .. I will let know how it comes out does parallel really work  to save time
